src/styles.scss

```diff
@@ -1 +1,17 @@
 /* You can add global styles to this file, and also import other style files */
```

Comment:
Remove this comment and add appropriate comment. That this style will have a global style and the other file imports.